Fix/some broken links (#225)
keymanapp/help.keyman.com: knowledge-base/kb0037.md

<p>If you experience this error, download the following batch file and run it to uninstall Keyman Desktop.  If your Program Files folder is not called "Program Files", you will need to edit the batch file and put the correct name for your Program Files folder (this may be the case on some non-English systems).</p>

<ul>

> <li>uninstkmp.zip - Uninstall Keyman Desktop Pro.</li>
> <li>uninstkml.zip - Uninstall Keyman Desktop Light.</li>

Reviewer: ```suggestion
<li>uninstkmp.zip is no longer available for download.</li>
<li>uninstkml.zip is no longer available for download.</li>
```
